stablehlo/dialect/CMakeLists.txt

```diff
@@ -172,7 +172,18 @@ add_mlir_dialect_library(VhloOps
 
   LINK_LIBS PUBLIC
   StablehloAssemblyFormat
+  VhloBytecode
   MLIRIR
   MLIRSupport
   MLIRQuantDialect
 )
+
+add_mlir_dialect_library(VhloBytecode
```

Comment:
Let's do the same thing as ChloBytecode.cpp and StablehloBytecode.cpp and merge this target into VhloOps?

Reply:
Makes sense to keep these together. `VhloBytecode.cpp` depends on `VhloOps.h` and `VhloOps.cpp` depends on `VhloBytecode.h`.